dotnet/src/SemanticKernel/SkillDefinition/SKFunction.cs

```diff
@@ -370,171 +327,6 @@ private void VerifyIsSemantic()
             "Invalid operation, the method requires a semantic function");
     }
 
-    // Run the semantic function
-    private async Task<SKContext> InvokeSemanticAsync(SKContext context, CompleteRequestSettings? settings)
-    {
-        this.VerifyIsSemantic();
-
-        this.EnsureContextHasSkills(context);
-
-        settings ??= this._aiRequestSettings;
-
-        var callable = (Func<ITextCompletion?, CompleteRequestSettings?, SKContext, Task<SKContext>>)this._function;
-        context.Variables.Update((await callable(this._aiService?.Value, settings, context).ConfigureAwait(false)).Variables);
-        return context;
-    }
-
-    // Run the native function
-    private async Task<SKContext> InvokeNativeAsync(SKContext context)
-    {
-        TraceFunctionTypeCall(this._delegateType, this._log);
-
-        this.EnsureContextHasSkills(context);
```

Comment:
I'll need to check if this is still required

Reply:
I didn't remove the lazy init. There was just a single caller and I moved the line to that location.